sklearn/linear_model/tests/test_ransac.py

```diff
@@ -90,12 +89,16 @@ def test_ransac_max_trials():
                                        random_state=0)
     assert_raises(ValueError, ransac_estimator.fit, X, y)
 
-    ransac_estimator = RANSACRegressor(base_estimator, min_samples=2,
-                                       residual_threshold=5, max_trials=11,
-                                       random_state=0)
-    assert getattr(ransac_estimator, 'n_trials_', None) is None
-    ransac_estimator.fit(X, y)
-    assert_equal(ransac_estimator.n_trials_, 2)
+    for i in range(50):
+        ransac_estimator = RANSACRegressor(base_estimator, min_samples=2,
```

Comment:
nitpick: You can move this to the outside and use `set_params` to avoid creating many objects.